CHANGELOG.md

```diff
@@ -8,15 +8,23 @@
 * Removed `slug` and `parent` fields.
 * Removed dependency upon `django-polymorphic-tree`.
 * Destroyed and recreated migrations.
+* Removed `conman.routes.handlers.SimpleHandler`.
+* Removed logic from `conman.routes.handlers.BaseHandler.handle`. Unless
+  overridden, this method will now raise a `NotImplementedError`.
 
 ### Added
 
 * Added support for django 1.10 (thanks to @Ian-Foote).
+* Added `RouteViewHandler`. A handler that delegates request handling to the
+  `view` attribute of the `Route`.
+* Added `URLConfHandler`. A handler that resolves a view based on its `urlconf`
+  attribute and the `path` provided.
```

Comment:
I think I should change this to look for `urlconf` on the `Route` instead of the `Handler` to be consistent with `RouteViewHandler`.

That's one for tomorrow, I think.